internal/gmm/message/build.go

```diff
@@ -854,11 +869,39 @@ func BuildConfigurationUpdateCommand(ue *context.AmfUe, anType models.AccessType
 		configurationUpdateCommand.LADNInformation.SetLADND(buf)
 	}
 
+	configurationUpdateCommand.ConfigurationUpdateIndication = nasType.
+		NewConfigurationUpdateIndication(nasMessage.ConfigurationUpdateCommandConfigurationUpdateIndicationType)
+	if configurationUpdateCommand.GUTI5G != nil ||
+		configurationUpdateCommand.TAIList != nil ||
+		configurationUpdateCommand.AllowedNSSAI != nil ||
+		configurationUpdateCommand.LADNInformation != nil ||
+		configurationUpdateCommand.ServiceAreaList != nil ||
+		configurationUpdateCommand.MICOIndication != nil ||
+		configurationUpdateCommand.ConfiguredNSSAI != nil ||
+		configurationUpdateCommand.RejectedNSSAI != nil ||
+		configurationUpdateCommand.NetworkSlicingIndication != nil ||
+		configurationUpdateCommand.OperatordefinedAccessCategoryDefinitions != nil ||
+		configurationUpdateCommand.SMSIndication != nil {
+		// TS 24.501 - 5.4.4.2 Generic UE configuration update procedure initiated by the network
+		// Acknowledgement shall be requested for all parameters except when only NITZ is included
+		configurationUpdateCommand.ConfigurationUpdateIndication.SetACK(uint8(1))
+		needTimer = true
+	}
+	if configurationUpdateCommand.MICOIndication != nil {
+		// Allowed NSSAI and Configured NSSAI are optional to request to perform the registration procedure
```

Comment:
I don't quite understand why allowed NSSAI and Configured NSSAI are excluded form this check since I see in TS 24.501 that
> The following parameters are sent to the UE with a request to perform the registration procedure for mobility and 
periodic registration update: 
a) MICO indication; 
b) UE radio capability ID deletion indication; and 
c) Additional configuration indication.
. Did I miss anything in spec?

Reply:
> The following parameters are sent to the UE with a request to perform the registration procedure for mobility and periodic registration update:
a) MICO indication;
b) UE radio capability ID deletion indication; and
c) Additional configuration indication.

I didn't see the description in TS 24.501 R15 but I saw it in TS 24.501 R16. Should we need to consider it since it is not the feature of Release 15?

Reply:
Sorry I didn't notice they're new for R16. Then please ignore those for now, thanks!